lib/src/key/mod.rs

```diff
@@ -50,29 +50,32 @@ pub mod bt; // Stores BTree nodes for terms
 pub mod bu; // Stores terms for term_ids
 pub mod database; // Stores the key prefix for all keys under a database
 pub mod db; // Stores a DEFINE DATABASE config definition
-pub mod dl; // Stores a DEFINE LOGIN ON DATABASE config definition
-pub mod dt; // Stores a DEFINE LOGIN ON DATABASE config definition
+pub mod dl; // Stores a DEFINE LOGIN ON DATABASE config definition (DEPRECATED)
+pub mod dt; // Stores a DEFINE TOKEN ON DATABASE config definition
+pub mod du; // Stores a DEFINE USER ON DATABASE config definition
 pub mod ev; // Stores a DEFINE EVENT config definition
 pub mod fc; // Stores a DEFINE FUNCTION config definition
 pub mod fd; // Stores a DEFINE FIELD config definition
 pub mod ft; // Stores a DEFINE TABLE AS config definition
 pub mod graph; // Stores a graph edge pointer
 pub mod index; // Stores an index entry
 pub mod ix; // Stores a DEFINE INDEX config definition
+pub mod ku;
 pub mod kv; // Stores the key prefix for all keys
 pub mod lq; // Stores a LIVE SELECT query definition on the database
 pub mod lv; // Stores a LIVE SELECT query definition on the table
 pub mod namespace; // Stores the key prefix for all keys under a namespace
-pub mod nl; // Stores a DEFINE LOGIN ON NAMESPACE config definition
+pub mod nl; // Stores a DEFINE LOGIN ON NAMESPACE config definition (DEPRECATED)
 pub mod ns; // Stores a DEFINE NAMESPACE config definition
 pub mod nt; // Stores a DEFINE TOKEN ON NAMESPACE config definition
+pub mod nu; // Stores a DEFINE USER ON NAMESPACE config definition
 pub mod pa; // Stores a DEFINE PARAM config definition
 pub mod sc; // Stores a DEFINE SCOPE config definition
 pub mod scope; // Stores the key prefix for all keys under a scope
 pub mod st; // Stores a DEFINE TOKEN ON SCOPE config definition
 pub mod table; // Stores the key prefix for all keys under a table
 pub mod tb; // Stores a DEFINE TABLE config definition
-pub mod thing;
+pub mod thing; // Stores a DEFINE USER ON KV config definition
```

Comment:
When I look at this in GitHub's interface, it seems like this line now reads `pub mod thing; // Stores a DEFINE USER ON KV config definition` which seems incorrect; isn't thing just a row in the database?

Reply:
Hmm, no, it's actually incorrect. I do remember putting the comment in the right place... I wonder if I messed up during a merge conflict :/ good catch!